rsciio/tests/test_trivista.py

```diff
@@ -0,0 +1,2373 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2022 The HyperSpy developers
+#
+# This file is part of HyperSpy.
```

Comment:
I just see it is still the old HyperSpy header, can you use the new RosettaSciIO header from one of the other files in the package in the newly contributed files.